community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/RemoveAcceptanceTest.scala

```diff
@@ -52,4 +52,52 @@ class RemoveAcceptanceTest extends ExecutionEngineFunSuite with QueryStatisticsT
     assertStats(result, labelsRemoved = 2)
     result.toList should equal(List(Map("labels(n)" -> List("L2"))))
   }
+
+  test("remove a single node property") {
```

Comment:
Perhaps add a test where the property does not exist, and a test where there are multiple nodes matching and running through.